app/javascript/components/GuitarGallery.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Title, TitleSizes, Gallery } from '@patternfly/react-core';
+import GuitarInfo from './GuitarInfo';
+import guitarsUrl from '../constUrls';
+import styled from 'styled-components';
+
+// is used in order to present a Loading 'screen' to the user until the data is rendered:
+const Loader = styled.div``;
```

Comment:
You can use the Spinner from Patternfly imho

Reply:
OK I'll read about it
Thanks @Ron-Lavi